packages/web-twig/tests/components-fixtures/helperText.twig

```diff
@@ -0,0 +1,18 @@
+<!-- DO NOT render without children -->
+<HelperText />
+
+<!-- Render with plain helper text -->
+<HelperText
+    className="myClass"
+    helperText="Helper text"
+    id="validationText"
```

Comment:
Maybe delete `className` and `id` here?

Reply:
Or add an example without both.

Reply:
I'm keeping the `className` because it's required.